### PR TITLE
Simple BuildInfo plugin

### PR DESCRIPTION
**Pull request by benjaminfrank**
Simple plugin to mimic sbt-buildinfo (https://github.com/sbt/sbt-buildinfo). Not remotely as fancy but so far worked without issues for us.
I will provide some tests in a later update.

### CODE REVIEW

**Comment by benjaminfrank** [comment]
build error on windows related to `Foo.java` and `Bar.java` which are not touched in this PR.
any advice welcome.

**Comment by benjaminfrank** [comment]
updated to be in a separate contrib folder and published as separate module.

**Comment by benjaminfrank** [comment]
added initial readme for plugins. it also mentions other available plugins. needs some polishing.

**Comment by benjaminfrank** [comment]
tests are also missing. I will add them later when I find a bit more time.
once we have sorted out how the `contrib` stuff will work properly I will also move the other plugins

**Comment by lihaoyi** [comment]
@benjaminfrank I've given you commit access. This looks good overall, so feel free to merge once you've got tests, the build is green and you're comfortable with the code

**Comment by benjaminfrank** [comment]
cool, thanks. it will take some time to polish as work load is ramping up as fast as family duties. but I am still committed to see those PRs merged one way or another 👍

**Comment by benjaminfrank** [comment]
Finally found some time to add simple tests. 
As so much time went by I need to validate the readme once more before this is good to go.

**Comment by benjaminfrank** [comment]
updated to master & moved documentation.
@lihaoyi @rockjam mind to have a quick look?

File: docs/pages/9 - Contrib Modules.md [hunk 1]
```diff
@@ -49,3 +49,61 @@ object example extends ScalaPBModule {
...
+        "scalaVersion" -> scalaVersion()
+      )
+    }
+  }
+  ```
+  
+#### Configuration options
+
+* `def buildInfoMembers: T[Map[String, String]]`  
+  The map containing all member names and values for the generated inffo object.
```
Comment (nightscape) [inline docs/pages/9 - Contrib Modules.md]: `inffo` -> `info`

**Comment by lihaoyi** [comment]
lgtm, feel free to merge

**Comment by benjaminfrank** [comment]
I do not understand why travis does have all those strange errors...

**Comment by benjaminfrank** [comment]
ok i found the issue: i changed some module definition

**Pull request merged**